Makefile

```diff
@@ -146,11 +146,33 @@ endif
 
 # setup opencv
 ifeq ($(USE_OPENCV), 1)
-	CFLAGS += -DMXNET_USE_OPENCV=1 $(shell pkg-config --cflags opencv)
-	LDFLAGS += $(filter-out -lopencv_ts, $(shell pkg-config --libs opencv))
+	CFLAGS += -DMXNET_USE_OPENCV=1
+	ifneq ($(USE_OPENCV_INC_PATH), NONE)
```

Comment:
This can be problematic when `USE_OPENCV_INC_PATH` is not defined in `config.mk`, because it's an empty string at the time. In that situation, no correct paths to headers and libs of opencv are actually added to `CFLAGS` and `LDFLAGS`.

Reply:
@reminisce good catch. @wkcn how about we add a default value for empty value here to help with cases where people already have old config.mk without the new variables?

Reply:
@reminisce @szha Hi! I have fixed it in the PR https://github.com/apache/incubator-mxnet/pull/14424